Makefile

```diff
@@ -125,7 +125,7 @@ build_integration: $(SOURCES)
 integration:
 GODOG_OPTS = --godog.tags=$(GOOS)
 ifndef PULL_SECRET_FILE
-	PULL_SECRET_FILE = --pull-secret-file=$(HOME)/Downloads/crc-pull-secret
+	PULL_SECRET_LOCATION = --pull-secret-file=$(HOME)/Downloads/crc-pull-secret
```

Comment:
Is it possible to start without this change? CI is broken with this. Or maybe handle both parameters.

Reply:
I left integration target as it was and added a `gintegration` target so we can control the two testsuites separately. 

I find `PULL_SECRET_FILE` vs `BUNDLE_LOCATION` inconsistent. But `--pull-secret-file` is an option for `crc start`... So, I left it as `FILE` everywhere for now.